common/pom.xml

```diff
@@ -22,6 +22,18 @@
   <build>
     <sourceDirectory>classes</sourceDirectory>
     <testSourceDirectory>tests</testSourceDirectory>
+    <resources>
+      <resource>
+        <directory>${project.basedir}/src/main/resources</directory>
+        <filtering>true</filtering>
+      </resource>
+    </resources>
+    <testResources>
+      <testResource>
+        <directory>${project.basedir}/src/test/resources</directory>
+        <filtering>true</filtering>
+      </testResource>
+    </testResources>
```

Comment:
Same as above

Reply:
There are `tests` that are using this `filter`

Reply:
Same as above